template.tex

```diff
@@ -1,12 +1,12 @@
 %% start of file `template.tex'.
-%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com).
+%% Copyright 2006-2015 Xavier Danaux (xdanaux@gmail.com), 2021 David Seus (cryptointerest@posteo.de).
 %
 % This work may be distributed and/or modified under the
 % conditions of the LaTeX Project Public License version 1.3c,
 % available at http://www.latex-project.org/lppl/.
 
 
-\documentclass[11pt,a4paper,sans]{./moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
+\documentclass[11pt,a4paper,sans]{moderncv}        % possible options include font size ('10pt', '11pt' and '12pt'), paper size ('a4paper', 'letterpaper', 'a5paper', 'legalpaper', 'executivepaper' and 'landscape') and font family ('sans' and 'roman')
```

Comment:
I'll probably have to double check this, but iirc I did this to prevent a the system moderncv to be used.

Reply:
as far as I know the file in the same directory takes precedence over the system installation. Latex complaints (warning) that the package provides the moderncv class and not ./moderncv class or similar. But we can also change it back if you prefer.

Reply:
Jup it works fine, thanks.